ebos/eclfluxmodule.hh

```diff
@@ -363,7 +363,7 @@ protected:
     {
         const auto& problem = elemCtx.problem();
 
-        bool enableBoundaryMassFlux = problem.hasFreeBoundaryConditions();
```

Comment:
nit: all decks have boundary conditions, they usually just happen to be trivial (i.e., no-flow)...

Reply:
DOne